react/src/CanvasElement.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { Chart } from './Chart';
+import { SpreadsheetWrapper as Spreadsheet } from './Spreadsheet';
+import { Element } from './Element';
+import { EmbedElement } from './rust_types/EmbedElement';
+import { BigNumber } from './components/BigNumber';
+
+export const CanvasElement = ({
+    element,
+    elementId,
+    dataHash,
+}: {
+    element: EmbedElement;
+    elementId: string;
+    dataHash: string;
+}) => {
+    if (!element) return <></>;
```

Comment:
type for `element` should be allow for undefined

Reply:
updated element prop type to allow undefined